buf.gen.yaml

```diff
@@ -5,7 +5,6 @@ managed:
     default: github.com/bufbuild/protoschema-plugins/internal/gen/proto
     except:
       - buf.build/bufbuild/protovalidate
-      - buf.build/liberetech/protoc-gen-bq-schema
```

Comment:
Bonus, we ended up publishing the module as well:

https://buf.build/googlecloudplatform/bq-schema-api